code/datums/gamemode/factions/bloodcult/bloodcult_runespells.dm

```diff
@@ -827,7 +832,7 @@
 		activator.client.images |= progbar
 
 	//secondly, let's stun our victim and begin the ritual
-	to_chat(victim, "<span class='danger'>Occult energies surge from below your feet and seep into your body.</span>")
+	to_chat(victim, "<span class='danger'>Occult energies surge from below your [issilicon(victim) ? "actuators" : "feet"] and seep into your [issilicon(victim) ? "chassis" : "body"].</span>")
```

Comment:
ableist towards borgs with no actuator and humans with no feet

Reply:
They're metaphorical actuators you bigot